.gitpod.Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM gitpod/workspace-full
+
+# Install custom tools, runtimes, etc.
+# For example "bastet", a command-line tetris clone:
+# RUN brew install bastet
+#
+# More information: https://www.gitpod.io/docs/config-docker/
```

Comment:
I guess it would be better if we provide tools for Libplanet developments here. you can find them [here](https://github.com/planetarium/libplanet/blob/master/CONTRIBUTING.md#prerequisites).

Reply:
Or, how about using `gitpod/workspace-dotnet`? [Gitpod seems to support .NET](https://www.gitpod.io/docs/languages/dotnet/).